bitnami/kubeapps/templates/apprepository/apprepositories.yaml

```diff
@@ -37,8 +39,11 @@ spec:
               value: {{ $.Values.apprepository.initialReposProxy.noProxy }}
       {{- end }}
       {{- if $.Values.apprepository.containerSecurityContext.enabled }}
-      securityContext:
-        runAsUser: {{ $.Values.apprepository.containerSecurityContext.runAsUser }}
+          securityContext: {{- omit $.Values.apprepository.containerSecurityContext "enabled" | toYaml | nindent 12 }}
+      {{- end }}
+      {{- end }}
+      {{- if $.Values.apprepository.podSecurityContext.enabled }}
+      securityContext: {{- omit $.Values.apprepository.podSecurityContext "enabled" | toYaml | nindent 8 }}
```

Comment:
Double check this change with PR from dev.

Reply:
This one is the correct change, as can be seen from Antonio's PR at https://github.com/vmware-tanzu/kubeapps/pull/6646/files#diff-879476c5d06fdfe1bc0c5c2d32d94f3b2a8d515b3faa4809623e2b283bdf9244

Reply:
Nice! Good catch